src/PolyType.Examples/CborSerializer/CborSerializer.Builder.cs

```diff
@@ -89,20 +89,20 @@ public CborConverter<T> GetOrAddConverter<T>(ITypeShape<T> typeShape) =>
                     new CborMutableEnumerableConverter<TEnumerable, TElement>(
                         elementConverter,
                         getEnumerable,
-                        enumerableShape.GetDefaultConstructor(),
+                        enumerableShape.GetDefaultConstructor(CollectionConstructionOptions<TElement>.Default),
```

Comment:
Could this be an optional parameter instead?

Reply:
I'd thought that making it nullable would involve an extra branch, but I see now that it's only during the visit pass, not during the execution of whatever the visit pass produces. I assume (benchmark level) perf is more critical for the second category of code than the first.